net.pokerth.PokerTH.yaml

```diff
@@ -0,0 +1,119 @@
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: pokerth
+rename-icon: pokerth
+rename-desktop-file: pokerth.desktop
+rename-metadata-file: pokerth.appdata.xml
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+
+modules:
+  - shared-modules/SDL/SDL-1.2.15.json
+  - shared-modules/SDL/SDL_mixer-1.2.12.json
+
+  - name: tinyxml
+    buildsystem: autotools
+    no-autogen: true
+    sources:
+      - type: archive
+        url: http://downloads.sourceforge.net/tinyxml/tinyxml_2_6_2.tar.gz
+        sha256: 15bdfdcec58a7da30adc87ac2b078e4417dbe5392f3afb719f9ba6d062645593
+      - type: file
+        path: tinyxml/configure.ac
+      - type: file
+        path: tinyxml/Makefile.am
+      - type: file
+        path: tinyxml/tinyxml.h.in
+      - type: shell
+        commands:
+          - autoreconf -fi
+    cleanup:
+      - /lib/*.la
+      - /include
+
+  - name: protobuf
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/protocolbuffers/protobuf/releases/download/v2.6.1/protobuf-2.6.1.tar.bz2
+        sha256: ee445612d544d885ae240ffbcbf9267faa9f593b7b101f21d58beceb92661910
+    cleanup:
+      - /lib/*.a
+      - /include
+
+  - name: gsasl
+    buildsystem: autotools
+    sources:
+    - type: archive
+      url: https://ftp.gnu.org/gnu/gsasl/gsasl-1.8.1.tar.gz
+      sha256: 4dda162a511976bfef14ad1fecd7733719dfd9c361b5d09dc8414ea9d472d8d2
+    cleanup:
+      - /bin
+      - /share/info
+
+  - name: libircclient
+    buildsystem: autotools
+    config-opts:
+      - --enable-shared
+      - --enable-ipv6
+      - --enable-openssl
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/libircclient/libircclient-1.10.tar.gz
+        sha256: bbb26f3af348b252c5204917a7f91cfdf172f1b6afbf4df1e561b03e20503c2d
+      - type: shell
+        commands:
+          - sed -i 's#$(DESTDIR)@libdir@#$(DESTDIR)@prefix@@libdir@#' src/Makefile.in
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -vfi
+    cleanup:
+      - /lib/*.la
+      - /include
+
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh --prefix=/app --with-libraries=thread,filesystem,date_time,program_options,iostreams,regex,random,chrono
+      - ./b2 -j $FLATPAK_BUILDER_N_JOBS
+      - ./b2 install
+    sources:
+      - type: archive
+        url: https://boostorg.jfrog.io/artifactory/main/release/1.65.0/source/boost_1_65_0.tar.bz2
+        sha256: ea26712742e2fb079c2a566a31f3266973b76e38222b9f88b387e3c8b2f9902c
+    cleanup:
+      - /lib/libboost_*.a
+      - /include
+
+  - name: simple
+    buildsystem: simple
+    build-commands:
+      - qmake PREFIX=/app CONFIG+="client" DEFINES+="BOOST_BIND_GLOBAL_PLACEHOLDERS" QMAKE_CFLAGS_ISYSTEM="" QMAKE_CXXFLAGS="-std=c++11" QMAKE_LIBDIR+="/app/lib" -spec linux-g++ pokerth.pro
```

Comment:
why doesn't this use `buildsystem: qmake` ?

Reply:
It never built `pokerth.pro` via the `config-options`.

Reply:
Actually, it does. Also managed to avoid a patch.